.github/workflows/deploy.yml

```
@@ -0,0 +1,56 @@
+  group: "pages"
+  cancel-in-progress: false
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Inject Soopr token
+        if: ${{ secrets.SOOPR_PUBLISH_TOKEN != '' }}
```

actionlint (1.7.9)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)